tough/src/error.rs

```diff
@@ -478,6 +478,46 @@ pub enum Error {
 
     #[snafu(display("Role missing from snapshot meta: {}", name))]
     RoleNotInMeta { name: String },
+
+    #[snafu(display("Not able to Sign message for key-id {} : {}", key_id, source))]
```

Comment:
No need to capitalize `Sign`.

Nit: single quotes around the key-id: `...for key-id '{}':...`